ada_2_fhir-r4/hg/0.2.0/sturen_verwijzing_of_update_huisarts_naar_paramedicus/fhir_instance/composition-hg-VerwijzingHuisartsParamedicusMaximaalbericht.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Composition xmlns="http://hl7.org/fhir">
+   <id value="composition-hg-VerwijzingHuisartsParamedicusMaximaalbericht"/>
+   <meta>
+      <profile value="http://nictiz.nl/fhir/StructureDefinition/hg-ReferralComposition"/>
+   </meta>
+   <identifier>
+      <value value="hg-VerwijzingHuisartsParamedicusMaximaalbericht"/>
+   </identifier>
+   <status value="final"/>
+   <type>
+      <coding>
+         <system value="http://loinc.org"/>
+         <code value="57133-1"/>
+         <display value="Referral note"/>
+      </coding>
+   </type>
+   <subject>
+      <reference value="Patient/patient-van-XXX-Sanderink-de-Kok"/>
+      <type value="Patient"/>
+      <display value="Patient, Sandra Janine (Sandra) van XXX_Sanderink"/>
+   </subject>
+   <date value="T"/>
+   <author>
+      <reference value="PractitionerRole/practitionerrole-000001111"/>
+      <type value="PractitionerRole"/>
+      <display value="Healthcare professional (role), Huisartsen, niet nader gespecificeerd, Gezondheidscentrum de Brug"/>
+   </author>
+   <title value="Referral note"/>
+   <section>
```

Comment:
The .text values are missing in the sections.

Reply:
Solved!